contentcuration/contentcuration/frontend/channelEdit/views/list/ContentNodeListItem.vue

```diff
@@ -0,0 +1,393 @@
+<template>
+
+  <li
+    class="content-list-item pa-0"
+    :nodeId="nodeId"
+    @dblclick="$router.push(isTopic ? treeLink : infoLink)"
+    @mouseenter="active = true"
+    @mouseover="active = true"
+    @mouseleave="active = false"
+  >
+    <VLayout
+      align-center
+      class="inline-list"
+      :class="[`${contentNode.kind}-kind`, {
+        'compact': isCompact,
+        'py-4': !isCompact,
+        'py-2': isCompact,
+      }]"
+    >
+      <div class="handle-col">
+        <transition name="fade">
+          <VBtn v-if="active" flat icon class="ma-0">
+            <Icon color="#686868">
+              drag_indicator
+            </Icon>
+          </VBtn>
+        </transition>
+      </div>
+      <div class="select-col mr-2">
+        <Checkbox v-model="checkboxSelected" class="mt-0 pt-0" />
+      </div>
+      <div
+        class="thumbnail-col py-2"
+        :class="{
+          'px-2': !isCompact,
+        }"
+      >
+        <figure
+          class="thumbnail"
+          :class="{
+            [contentNode.kind]: isCompact,
+            'icon': !showThumbnail,
+            'icon-only': isCompact,
+          }"
+        >
+          <VLayout
+            v-show="showThumbnail"
+            tag="figcaption"
+            row
+            align-center
+            :class="contentNode.kind"
+          >
+            <VFlex shrink class="pr-1">
+              <Icon
+                v-if="showThumbnail"
+                dark
+                small
+                :aria-label="kindTitle"
+                v-text="icon"
+              />
+            </VFlex>
+            <VFlex shrink>
+              <span class="white--text caption">{{ kindTitle }}</span>
+            </VFlex>
+          </VLayout>
+          <img
```

Comment:
I did add a [thumbnail component here](https://github.com/learningequality/studio/blob/catalog/contentcuration/contentcuration/frontend/shared/views/files/Thumbnail.vue), so you can just pass in the src/encoding and it will handle the aspect ratios and styling for you

Reply:
Is there an example of that? This is highly responsive.

Reply:
It's used in the [move modal ](https://github.com/learningequality/studio/blob/vue-refactor/contentcuration/contentcuration/frontend/channelEdit/move/MoveModal.vue#L76). Since the styling is the same, do you mind updating the shared component to match yours so it can be used across the app?